Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM registry.access.redhat.com/ubi8/ubi
+FROM registry.access.redhat.com/ubi9/ubi-minimal
```

Comment:
RedHat backports patches to their distros most container scanners don't recognize them so switch to ubi9 for newer base versions.

Reply:
that's an interesting note about ubi9. maybe IQ should make the jump too, since they're back on ubi8-minimal.

Reply:
We decided for the moment to limit this to ubi8 and may switch to ubi9 later